lib/Fetcher/Client/FeedIoClient.php

```diff
@@ -45,9 +45,10 @@ public function __construct(\GuzzleHttp\ClientInterface $guzzleClient)
     public function getResponse(string $url, \DateTime $modifiedSince) : ResponseInterface
     {
         try {
+            $modifiedSince->setTimezone(new \DateTimeZone('GMT'));
```

Comment:
I think that does not belong into the try-catch-block.

Reply:
You mean move it outside, and before?

Reply:
It should be fixed now. Don't really know what I was doing there :S